src/Symfony/Component/HttpKernel/Tests/EventListener/SaveSessionListenerTest.php

```diff
@@ -45,9 +45,5 @@ public function testSessionSavedAndResponsePrivate()
         $request->setSession($session);
```

Comment:
The test method can now be named `testSessionSaved()` instead of `testSessionSavedAndResponsePrivate()`?

Reply:
fixed